src/utils/fetchClient.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-const BASE_URL = 'https://mate.academy/students-api';
+const BASE_URL = 'https://mate.academy/students-apif';
```

Comment:
```suggestion
const BASE_URL = 'https://mate.academy/students-api';
```
why do you need this "f" letter here?